content/java/concepts/strings/terms/startsWith/startsWith.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.startsWith()'
+Description: 'Returns true if a string starts with given character(s) and false otherwise.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Booleans'
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.startsWith()`** method returns `true` if a string starts with given character(s). Otherwise, `false` is returned.
+
+## Syntax
+
+```pseudo
+string.startsWith(object);
+```
+
+The `object` can either be a string literal or a representation of a `String` value. This will return `true` if the `object` has the same starting character sequence as `string` .
```

Comment:
This sentence is a bit confusing. Can you reword it? Also, `object` is now `stringOfCharacters`.